edi_oca/models/edi_exchange_type.py

```diff
@@ -50,7 +50,25 @@ class EDIExchangeType(models.Model):
     direction = fields.Selection(
         selection=[("input", "Input"), ("output", "Output")], required=True
     )
-    exchange_filename_pattern = fields.Char(default="{record_name}-{type.code}-{dt}")
+    exchange_filename_pattern = fields.Char(
```

Comment:
I think the help should be fixed in `edi_storage_oca` instead, can you prepare a PR on https://github.com/QuocDuong1306/edi-framework/pull/5 to change it there?